### PR TITLE
New PSSE models (COMP, ES, TG, UEL, Machines)

### PR DESCRIPTION
**Pull request by GiuseppeLaera**
New control models from PSSE library have been implemented

### CODE REVIEW

**Comment by lvanfretti** [comment]
@marcelofcastro I talked to @GiuseppeLaera to get the models form the Dominion project into the library.
They need some cleaning up, can you please review?
If you require changes, please assign to both @GiuseppeLaera and @lvanfretti so that one of us takes care of this.
I need this to be merged so I can also add the manual implementation and automated implementation of several plants that we made and should be put under the "Examples" sub-package.

**Comment by dietmarw** [comment]
This PR is introducing 4 (!) new PI(D) controllers. Please argue very carefully why this is needed and why `Modelica.Blocks.Continuous.LimPID` can not be used instead.

**Comment by lvanfretti** [comment]
@GiuseppeLaera can you please give us an update on the progress? We were targeting to prepare a release on Oct. 31st... but we can't move forward with #341 without the changes needed here.

**Comment by GiuseppeLaera** [comment]
@lvanfretti I am trying to be done with the updates of the models by the end of next week (10th of November)

**Comment by lvanfretti** [comment]
@GiuseppeLaera thanks for letting us know, how are the updates coming along?

**Comment by GiuseppeLaera** [comment]
I have just made a new commit

**Comment by lvanfretti** [comment]
> I have just made a new commit

Thanks @GiuseppeLaera, really appreciate it!
@fachif please keep an eye on the progress of this PR so you can finish up with yours.

**Review by dietmarw — CHANGES_REQUESTED** [review]
I've done a couple of clean up commits so that the new contribution follows our code conventions. Please looks also at the inline comments, especially regarding the changes to HYGOV

**Comment by dietmarw** [comment]
@GiuseppeLaera I've implemented the new PI(D) controllers in a more Modelica like fashion see: https://github.com/OpenIPSL/OpenIPSL/pull/347/commits/ecd0f280a9fe6eb574e038158d5ad68b524bd731 
Doing so let me wonder why you are using `KDR` in some models but `K_DR` in others same for the `KIR` and `K_IR`. That seems a bit messy. Could this be unified?

File: OpenIPSL/Electrical/Controls/PSSE/TG/HYGOV.mo [hunk 1]
```diff
@@ -152,11 +153,7 @@ equation
           extent={{-100,160},{100,100}},
           lineColor={28,108,200},
           textString="HYGOV")}),
-    Documentation(info="<html> 
-<p> This is a hydro turbine-governor model. </p>
-<p> If compared to the original PSSE implementation one will notice that the initial water head value, <code>h0</code>, is declared as a parameter.
-This allows the user to modify it, if required, in order to perform adequate studies. 
-The user should leave it in its default value (<code>1</code>) if they do not have any information on water head value, or if they do not wish to modify the parameter.</p>
+    Documentation(info="<html> This is a hydro turbine-governor model.
 </html>",revisions="<html>
```
[the remark this answers is not in the document]
Comment (GiuseppeLaera) [reply OpenIPSL/Electrical/Controls/PSSE/TG/HYGOV.mo:157]: You can disregard the changes to HYGOV introduced by my previous commit

File: OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/WEHGOV.mo [hunk 2]
```diff
@@ -0,0 +1,464 @@
+within OpenIPSL.Electrical.Controls.PSSE.TG.BaseClasses;
+package WEHGOV
+
```
[the remark this answers is not in the document]
Comment (GiuseppeLaera) [reply OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/WEHGOV.mo]: Just made a new commit to fix it

File: OpenIPSL/Electrical/Controls/PSSE/UEL/package.mo [hunk 3]
```diff
@@ -0,0 +1,3 @@
+within OpenIPSL.Electrical.Controls.PSSE;
+package UEL
```
[the remark this answers is not in the document]
Comment (GiuseppeLaera) [reply OpenIPSL/Electrical/Controls/PSSE/UEL/package.mo]: Just made a new commit to fix it

**Comment by GiuseppeLaera** [comment]
@dietmarw the confusion about the PI(D) parameters naming comes from the different conventions used in PSSE manuals ("ModelLibrary.pdf" and "PAGV2.pdf"). So the unification is better. Thanks!

**Comment by dietmarw** [comment]
Ok but still you need to unify them currently both are used one type in AC7B and the other in AC8B. Can this be cleaned up?

**Review by dietmarw — APPROVED** [review]
Looks fine now for me. 
@marcelofcastro  any objections to merge it?

**Review by marcelofcastro — APPROVED** [review]
No objections, @dietmarw and @GiuseppeLaera.

**Pull request merged**